### ISSUE TITLE
Regarding Table 1 results in the paper

### ISSUE THREAD

**Issue by HashmatShadab** [jiamingzhang94/Adversarial-Prompt-Tuning#3]
Hi, Thanks for sharing your work. I just needed some clarity in the experiments done to report results in Table 1.

1. Vanilla CLIP reports the ZS clean and adversarial performance on the datasets reported in Table 1?
2. AdvPT based CLIP is adversarially trained on each of the mentioned datasets(Flowers, Pets,...) and the adversarial performance is reported? 
3. Also regarding the adversarial examples generated for evaluation, Are they generated with the same objective as that for generating adversarial embedding bank?

So, if i understand correctly, results for ZS adversarial robustness(Flowers, Pets,...) are not evaluated?

**Comment by jiamingzhang94** [comment]
Hi, 
Could you please clarify what you mean by “results for ZS adversarial robustness”? I am not sure about the difference you are referring to between "ZS adversarial performance" and "results for ZS adversarial robustness."

**Comment by HashmatShadab** [comment]
What i mean by Zero shot adversarial robustness is similar to how it mentioned in the paper titled "Understanding Zero-Shot Adversarial Robustness for Large-Scale Models". They adversarially finetune CLIP using ImageNet. Then evaluate the robust CLIP on adversarial examples crafted on downstream datsets on which it has not been adversarially trained.

**Comment by jiamingzhang94** [comment]
If I understand correctly, you are referring to: performing AT on CLIP using ImageNet, and then testing robustness on the test sets of downstream datasets (adversarial versions of the test sets). This scenario is not within the scope of our paper because this setup is more stringent (it requires the model to not use the training sets of downstream datasets for fine-tuning). In fact, you can observe that the results of TeCoA are much lower compared to AdvPT.

**Comment by HashmatShadab** [comment]
Thank you for clarifying.

**Comment by HashmatShadab** [comment]
The comparison with Vanilla CLIP results in Table 1, is vanilla CLIP also fine tuned on downstream datasets or is it just pretrained CLIP model?

**Comment by jiamingzhang94** [comment]
Pretrained CLIP model. If you are looking for comparisons with other methods that also utilize downstream datasets, you may refer to Figure 3 (CoOp) and Linear Prob (in the camera-ready version) for relevant results.

**Comment by HashmatShadab** [comment]
@jiamingzhang94  Regarding the vanilla CLIP model results in Table 1. The vanilla CLIP model is showing significant robustness even though it does not have the prompts optimized in the text encoder. From what i understand for vanilla CLIP results, you are evaluating zero shot clean and adversarial performance on multiple datasets. If that's true, what is the reason for this robustness. There are several works which show, specifically on  ViT-L/14 CLIP, that its very vulnerable to attacks at very low perturbation budgets (eps=2/255).

![image](https://github.com/user-attachments/assets/8419b50f-3264-4350-82e3-32000efcd0ba)
The above figure shows what I am trying to say. While i understand there method focuses on zero-shot adversarial robustness which is not the goal of the this work, but don't you use the vanilla CLIP results in a similar fashion? Kindly let me know where I might have misunderstood the results from Table 1 (specifically for vanilla CLIP).

**Comment by jiamingzhang94** [comment]
I am occupied at the end of the year, and I have not read this paper you mentioned. So you will need to check the original text and code yourself to compare the differences in settings, such as the attack method (PGD), the number of iterations (40), and the adversarial loss (Kullback-Leibler Divergence).

**Comment by HashmatShadab** [comment]
Thank you for your time. Can you just confirm that vanilla clip is being evaluated on multiple datasets in table 1 in zero shot manner. No finetuning.

**Comment by jiamingzhang94** [comment]
zero shot manner

**Comment by HashmatShadab** [comment]
Thank you!
